src/makeRobotAccountant.js

```diff
@@ -5,8 +5,18 @@
  * @return {function}
  */
 
-function makeRobotAccountant() {
-  // write code here
+function makeRobotAccountant(a) {
```

Comment:
The makeRobotAccountant function should not take any arguments, please remove the unnecessary parameter 'a'.